lms/envs/common.py

```diff
@@ -1329,6 +1333,18 @@
         ],
         'output_filename': 'css/lms-style-xmodule-annotations.css',
     },
+    'style-edx-footer': {
+        'source_filenames': [
+            'sass/footer-v3.css',
+        ],
+        'output_filename': 'css/lms-style-edx-footer-v3.css',
+    },
+    'style-edx-footer-rtl': {
+        'source_filenames': [
+            'sass/footer-v3-rtl.css',
+        ],
+        'output_filename': 'css/lms-style-edx-footer-v3.css',
```

Comment:
Mind syncing up the ``output_filename`` value to match consistency with what we've [moved to for the LMS main CSS](https://github.com/edx/edx-platform/blob/alasdair/ECOM-1547-footer-2015/lms/envs/common.py#L1283)? 

So, something like ``'output_filename': '/css/lms-footer-edx.css,``is what I'd expect.